src/firmware/linux/host/ioctl.rs

```diff
@@ -24,6 +24,8 @@ impl_const_id! {
     GetId<'_> = 8, /* GET_ID2 is 8, the deprecated GET_ID ioctl is 7 */
 
     SnpPlatformStatus = 9,
+    SnpSetExtConfig = 0xA,
+    SnpGetExtConfig = 0xB,
```

Comment:
Nitpick: For consistency, I think either those new values should be specified as decimal, or the already present ones moved to hexadecimal.

Reply:
I switched them all to hex for consistency.